test/unit/algo/converting_eve_iterator.cpp

```diff
@@ -1,14 +1,6 @@
-//==================================================================================================
```

Comment:
why the name change for the files ?

Reply:
This one: because I don't test the non-eve iterator in this file. 
There is also now a convert.cpp to test the rest of the logic.